numba/tests/test_parfors.py

```diff
@@ -1839,6 +1839,14 @@ def test_impl(a):
         x = np.ones((3,3))
         self.check(test_impl, x)
 
+    @skip_parfors_unsupported
+    def test_high_dimension1(self):
+        # issue6749
+        def test_impl(x):
+            return x * 1.0
```

Comment:
```suggestion
            return x * 5.0
```
Suggest altering the multiplier given the input is all `1.0` else there's no differentiation between it working and a NOP?